catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -81,11 +96,13 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DumpCommand.class);
 
-  private static final String ZIP_COMPRESSION = "zipCompression";
-
   private List<MetacardTransformer> transformers = null;
 
-  @VisibleForTesting volatile Optional<QueryResponseTransformer> zipCompression = Optional.empty();
+  private static final String METACARD_PATH = "metacards" + File.separator;
+
+  private static final String CONTENT = "content";
+
+  private static final int BUFFER_SIZE = 10000000;
```

Comment:
✏️ Just for future reference, if you want to delineate large numbers java allows you to insert `_` into numbers as separator characters, eg `BUFFER_SIZE = 10_000_000`